beamer/tests/agent/test_challenge.py

```diff
@@ -344,3 +344,72 @@ def test_withdraw_not_participant(request_manager, token, config):
     agent.wait()
 
     assert withdraw_event is None
+
+
+# Scenario 7:
+#
+# Bob              Charlie              Dave
+# -----------------------------------------------
+#                  claim
+#                                       challenge
+#                  challenge
+# challenge
+# Winner: Bob + Dave
+#
+# Note: testing that the agent can handle multiparty bidding
+def test_challenge_7(request_manager, fill_manager, token, config):
+    target_address = accounts[8]
+    requester, charlie, dave = accounts[:3]
```

Comment:
We should use `alloc_accounts` now that it is available also to agent tests.

Reply:
let's do this in a different PR, as it counts for all tests